### PR TITLE
WebApp initial translation setup

### PR DESCRIPTION
**Pull request by raix**
### Summary & Motivation

Add translations setup in WebApp using Lingui. Add `LOCALE` to the runtime environment from the server request culture.

Add translations to pages with `en-US` and `da-DK` translation. Translations are type-safe and load dynamically as needed.

The build responsibility for the WebApp has been moved to the web app project. Removing the duplicate build of the WebApp speeds up the build.

Note that Zod schema validation messages are not currently translated - waiting for API improvements in the Zod project.

For now, `swc-loader` is used instead of the `builtin:swc-loader` due to a lack of support for plugins - this is on the Rspack roadmap - for now, build time is degraded.

### Checklist

- [x] I have added a Label to the pull-request
- [x] I have added tests, and done manual regression tests
- [x] I have updated the documentation, if necessary

### CODE REVIEW

File: application/account-management/WebApp/src/app/page.tsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
+import { Trans, t } from "@lingui/macro";
+import { LocaleSwitcher } from "@/ui/LocaleSwitcher";
+
 export default function Page() {
   return (
     <div>
-      <h2>Root</h2>
-      <p>This is the main page</p>
+      <h2>
+        <Trans>Overview</Trans>
```
Comment (tjementum) [inline application/account-management/WebApp/src/app/page.tsx:8]: Is `<Trans>` our own html tag. Anyway, we can call it `<translation>` (I guess we also want it lowercase?

File: application/account-management/WebApp/src/main.tsx [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+
```
Comment (tjementum) [inline application/account-management/WebApp/src/main.tsx:3]: Is this on purpose?

File: application/account-management/WebApp/src/translations/locale/da-DK.po [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+"Language: da-DK\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+msgid "Overview"
+msgstr "Oversigt"
```
Comment (tjementum) [inline application/account-management/WebApp/src/translations/locale/da-DK.po:17]: Are `msgid` and `msgstr` our own. Can we avoid the abbrivations?

**Comment by tjementum** [comment]
Not sure why the build fails?

Please add more description to the pull-request. You can use [this GPT that will help you, just write a rough description describing the changes](https://chat.openai.com/g/g-u5NpNgL9t-platformplatform-pull-request).

File: application/account-management/WebApp/src/translations/locale/da-DK.po (hunk 3 above)
Comment (raix) [reply application/account-management/WebApp/src/translations/locale/da-DK.po:17]: it's part of the format https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html

File: application/account-management/WebApp/src/app/page.tsx (hunk 1 above)
Comment (raix) [reply application/account-management/WebApp/src/app/page.tsx:8]: It's from the lingui package - would personally prefer:
```
t`Overview`
```

We could create an alias but developers reading the library docs might find it confusing

File: application/account-management/WebApp/src/translations/locale/da-DK.po [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+msgstr "Kontoadministration"
+
+msgid "Create a account"
+msgstr "Opret en konto"
+
+msgid "Create Account"
+msgstr ""
+
+msgid "Create account!"
+msgstr ""
```
Comment (tjementum) [inline application/account-management/WebApp/src/translations/locale/da-DK.po:46]: Missing translations

File: application/account-management/WebApp/src/translations/locale/pseudo.po
Comment (tjementum) [inline application/account-management/WebApp/src/translations/locale/pseudo.po:1]: What is this file. Seems like we have to manually keep the translations in sync? Or is there something automated?

**Review by tjementum — DISMISSED** [review]
Looks good now, and everything works. There are a couple SonarCloud findings for the Frontend, that is block-in the merge.

**Comment by tjementum** [comment]
BTW.... Please create an issue with the `enhancement` tag on the Kanban board and link to this pull request and link that issue to the `Roadmap issue` on the roadmap board.

**Review by tjementum — APPROVED** [review]

**Pull request merged**